variables.tf

```diff
@@ -194,3 +194,8 @@ variable "platform_node_type" {
   type    = string
   default = "n1-standard-8"
 }
+
+variable "platform_namespace" {
+  type    = string
+  default = "domino-platform"
+}
```

Comment:
Is this used in the deployer or just leftover from testing?

Reply:
ahh just leftover, removed!